packages/docs/nuxt.config.ts

```diff
@@ -159,6 +159,7 @@ export default defineNuxtConfig({
   runtimeConfig: {
     public: {
       VITE_STORYBOOK_HOSTNAME: process.env.STORYBOOK_HOSTNAME,
+      recaptchaKey: process.env.NUXT_PUBLIC_RECAPTCHA_KEY,
```

Comment:
Maybe we can use the same syntax in all cases?

i.e. `NUXT_PUBLIC_RECAPTCHA_KEY`

Right now - there are 3 ways:
* NUXT_PUBLIC_RECAPTCHA_KEY
* RECAPTCHA_SITE_KEY
* recaptchaKey

Reply:
Done